pkg/transset

```diff
@@ -0,0 +1,25 @@
+[mirrors]
+https://gitlab.freedesktop.org/xorg/app/transset/-/archive/transset-1.0.3/transset-transset-1.0.3.tar.bz2
+
+[vars]
+filesize=10891
+sha512=b2b64e273601727ba7188604894730377322ead7b0c8d4465aa44256aad1518a3d76f9c19d6b93e3fde3ca4628eb82310618701f3492565012e7c44631a70e46
+pkgver=1
+
+[deps.host]
+autoconf
+
+[deps.host]
+autoconf
+
+[build]
+autoreconf -vif
+
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
+
+LDFLAGS="-Wl,-z,lazy" \
```

Comment:
please use KEEP/pkg_skel/autoconf template steps for configure, make, make install

otherwise user-provided CFLAGS won't be used

you can add lazy to the LDFLAGS defined there